src/pymmcore_plus/core/_sequencing.py

```diff
@@ -277,6 +278,21 @@ def _nope(reason: str) -> tuple[bool, str] | bool:
     elif cur_length >= core.getExposureSequenceMaxLength(cam_dev):  # pragma: no cover
         return _nope(f"Camera {cam_dev!r} {max_len=} < {cur_length=}")
 
+    # SLM
+    try:
+        if e1.slm_image != e2.slm_image:
```

Comment:
ah, we could also make this easier in useq_schema as well.  one should be able to do an equality check without fear :). will make issue

Reply:
fearless equality check implemented and tested! works with updated version of useq.